src/mem/mem.c

```diff
@@ -4,6 +4,8 @@
  * Copyright (C) 2010 Creytiv.com
  */
 #include <ctype.h>
+#include <stddef.h>
+#include <stdint.h>
```

Comment:
Should not be needed for uint32_t (provided by re_types.h)

Reply:
Ok, will remove.

I always include all headers that are needed by a given file, relying on includes made by other headers is nothing but trouble.

Reply:
We try to include only needed headers and have to rely on <re_types.h> for portability.